### DISCUSSION TITLE
Welcome to GoTch Discussion!

### DISCUSSION THREAD

**Discussion by sugarme** [sugarme/gotch#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by manjunathsc24** [comment]
Hi sugarme
when i am installing gotch

i am getting below error , could you please let me know how to solve , i have followed all the steps mentioned but still getting the below error. it will be great helpful for me if you able to solve this issue.

go get "github.com/sugarme/gotch"
# github.com/sugarme/gotch/libtch
torch_api.cpp:1:9: fatal error: 'torch/csrc/autograd/engine.h' file not found


thank you
Manjunath
